modules/tracker/controllers/components/ApisubmissionComponent.php

```diff
@@ -185,8 +228,13 @@ public function post($args)
      * @path /tracker/submission/{id}
      * @http PUT
      * @param id
-     * @param uuid (Optional)
      * @param name (Optional)
+     * @param submitTime (Optional)
+     * @param branch (Optional)
+     * @param buildResultsUrl (Optional)
+     * @param params (Optional)
```

Comment:
I don't think `params` would be saved correctly just setting them on the DAO.

Probably if `params` is passed, the existing params connected to this submission should be removed and the new params saved.